lib/reversi_methods.rb

```diff
@@ -46,8 +46,9 @@ def put_stone(board, cell_ref, stone_color, dry_run: false)
     raise 'すでに石が置かれています' unless pos.stone_color(board) == BLANK_CELL
 
     # コピーした盤面にて石の配置を試みて、成功すれば反映する
+    #boardに記録された文字列をオブジェクトに、オブジェクトを文字列へ変更して変数に代入
```

Comment:
この行もバグ修正として必要な修正か？というとなんか違う気がしますね 🤔 
あと、コメントを書くにしても

```suggestion
    # boardに記録された文字列をオブジェクトに、オブジェクトを文字列へ変更して変数に代入
```

というようにスペースを入れることが多いです